src/components/Quote.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
+import { useEffect, useState } from 'react';
+import Loader from './Loader';
+import Error from './Error';
+
+const Quote = () => {
+  const [quote, setQuote] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  // Fetch data
+  const fetchData = async () => {
+    const request = await fetch(
+      'https://api.api-ninjas.com/v1/quotes?category=age',
+      {
+        method: 'Get',
+        contentType: 'application/json',
+        headers: { 'X-Api-Key': 'cwd7G46iynGqOEQ1g+zBbw==5ezDbVZRFlCVb0uD' },
+      }
+    );
+    const response = await request.json();
+    // Error
+    if (response.error) {
+      setLoading(false);
+      setError(response);
+    } else {
+      // normal
+      setLoading(false);
+      setError(false);
+      setQuote(response[0]);
+    }
```

Comment:
- [ ] [OPTIONAL] Most times it is good to make your API calls within a try-catch block, this is good for error handling you can try it out :+1: